requirements-test.txt

```diff
@@ -3,6 +3,7 @@ numpy
 pulp
 pytest >= 8.1
 expecttest
+psutil
```

Comment:
https://github.com/pytorch/pytorch/pull/169613 This PR introduced this import that is breaking the Auto-Parallel CI